export_html_text/models/html_field_text_config.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HtmlFieldTextConfig(models.Model):
```

Comment:
In stead of this, I thought it would make a better design if we had an "Export HTML in Text" checkbox in the export dialog. Can you please consider this approach?

<img width="1351" height="246" alt="image" src="https://github.com/user-attachments/assets/ac0a3419-a59b-4320-b80e-260a9ffa39b8" />